scene/gui/tree.cpp

```diff
@@ -2625,6 +2625,61 @@ int Tree::draw_item(const Point2i &p_pos, const Point2 &p_draw_ofs, const Size2
 	return htotal;
 }
 
+void Tree::draw_background_stripes(const Point2 &draw_ofs) {
+	const RID ci = get_canvas_item();
+
+	int stripe_count = 0;
+	TreeItem *current_item = root;
+	int total_size = draw_ofs.y - theme_cache.v_separation;
+
+	if(_get_title_button_height() > 0) {
+		stripe_count++;
+	}
+
+	// Stripes per item, respects item height.
+	while (current_item) {
+		if (const int item_height = compute_item_height(current_item); item_height > 0) {
```

Comment:
```suggestion
		const int item_height = compute_item_height(current_item);
		if (item_height > 0) {
```
We don't generally allow if statements like this, they hurt readability significantly